openedx/features/student_certificates/helpers.py

```diff
@@ -197,6 +197,10 @@ def get_pdfkit_options(image_width, image_height):
     :param image_height: image height in pixel
     :return: pdfkit options dict
     """
+    # convert pixel to centimeter
+    image_width = image_width * 2.54 / 96
+    image_height = image_height * 2.54 / 96
+
```

Comment:
can we remove this comment, and instead create a method `convert_pixels_to_centimeters`?

And the new variable name should be different, like `image_width_in_pixels = convert_pixels_to_centimeters(image_with)`